v2/client_integration_test.go

```diff
@@ -0,0 +1,29 @@
+package binance
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+)
+
+type baseIntegrationTestSuite struct {
+	suite.Suite
+	client *Client
+}
+
+func SetupTest(t *testing.T) *baseIntegrationTestSuite {
+	apiKey := os.Getenv("BINANCE_API_KEY")
+	secretKey := os.Getenv("BINANCE_SECRET_KEY")
+
+	if apiKey == "" || secretKey == "" {
```

Comment:
@pcriadoperez shouldn't we also support the testnet?

Reply:
Good point, I added to the portfolio integration client and added also the option for the proxy.